### DISCUSSION TITLE
Code optimization

### DISCUSSION THREAD

**Discussion by CWillberg** [PeriHub/PeriLab.jl#161] [General]
In "heat_transfer.jl" I think   floor(specific_volume[iID]) can be deleted. I think the specific volume can be between 0 and 1. and floor will activate and deactivate the sum

  if surface_nodes[iID] && specific_volume[iID] > req_specific_volume
      heat_flow[iID] += (kappa * (temperature[iID] - Tenv)) / dx * floor(specific_volume[iID])
    end

**Comment by JTHesse** [comment]
I added this because the specific volume for surface nodes is always greater than 1.0 (mostly between 1.5 and 1.8), but as we want to calculate the surface area, we need to use the rounded down specific volume. Otherwise, the heat flow will be too high.
It's not really intuitive but working well 😄
